java/line-of-sight-geoelement/README.md

```diff
@@ -1,28 +1,34 @@
-# Line of Sight GeoElement
+# Line of sight geoelement
+
 Show a line of sight between two moving objects.
 
-## Use Case
-To determine if an observer can see a target, you can show a line of sight between them. The line will be green until it is obstructed, in which case it will turn red. By using the GeoElement variant of the line of sight, the line will automatically update when either GeoElement moves.
+![Image of line of sight geoelement](line-of-sight-geoelement.png)
+
+## Use case
+
+A line of sight between `GeoElement`s (i.e. observer and target) will not remain constant whilst one or both are on the move.
 
-![Line of Sight GeoElement App](line-of-sight-geoelement.png)
+A `GeoElementLineOfSight` is therefore useful in cases where visibility between two `GeoElement`s requires monitoring over a period of time in a partially obstructed field of view
+(such as buildings in a city).
 
 ## How to use the sample
 
-A line of sight will display between a point on the Empire State Building (observer) and a taxi (target). The taxi will drive around a block and the line of sight should automatically update. The taxi will be highlighted when it is visibile. You can change the observer height with the slider to see how it affects the target's visibility.
+A line of sight will display between a point on the Empire State Building (observer) and a taxi (target).
+The taxi will drive around a block and the line of sight should automatically update.
+The taxi will be highlighted when it is visible. You can change the observer height with the slider to see how it affects the target's visibility.
```

Comment:
It looks like the taxi isn't just highlighted, instead it has a flashing highlight? And the main thing is that the line of sight is red for the portion that is obscured.